test/hotspot/jtreg/runtime/logging/FoldMultilinesTest.java

```diff
@@ -65,27 +79,33 @@ private static void analyzeFoldMultilinesInvalid(ProcessBuilder pb) throws Excep
         output.shouldNotHaveExitValue(0);
     }
 
-    public static void main(String[] args) throws Exception {
+    private static void test(String out) throws Exception {
         String Xlog;
         ProcessBuilder pb;
 
-        Xlog = XLOG_BASE + "::foldmultilines=true";
+        Xlog = XLOG_BASE + out +  "::foldmultilines=true";
         pb = ProcessTools.createJavaProcessBuilder(Xlog, InternalClass.class.getName());
-        analyzeFoldMultilinesOn(pb);
+        analyzeFoldMultilinesOn(pb, out);
 
-        Xlog = XLOG_BASE + "::foldmultilines=false";
+        Xlog = XLOG_BASE + out + "::foldmultilines=false";
         pb = ProcessTools.createJavaProcessBuilder(Xlog, InternalClass.class.getName());
-        analyzeFoldMultilinesOff(pb);
+        analyzeFoldMultilinesOff(pb, out);
 
-        Xlog = XLOG_BASE + "::foldmultilines=invalid";
+        Xlog = XLOG_BASE + out + "::foldmultilines=invalid";
         pb = ProcessTools.createJavaProcessBuilder(Xlog, InternalClass.class.getName());
         analyzeFoldMultilinesInvalid(pb);
     }
 
+    public static void main(String[] args) throws Exception {
+        test("file=" + EXCEPTION_LOG_FILE);
+        test("stdout");
+        test("stderr");
```

Comment:
I think we should add a case whe the output is not explicitly specified. I.e.,

```
-Xlog:exceptions=info:::foldmultilines=true
```

Reply:
I missed to check the log if output is default ("") and foldmultilines=false. So I refactored testcase.